api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -747,6 +747,17 @@ public Long getCountOfPatients(String query, boolean includeVoided) {
 	}
 	
 	public List<Patient> findPatients(String query, boolean includeVoided, Integer start, Integer length){
+		//Just for testing...It has nothing to do with the function
+			List<String> testfields=new ArrayList<>();
+			testfields.add("gender");
+			LuceneQuery<Person> testQuery=LuceneQuery.newQuery(Person.class, sessionFactory.getCurrentSession(), "M", testfields);
```

Comment:
can u try with already existing fields and see if this way of running the lucene query would work for any of them!